generators/app/templates/documentation.js

```diff
@@ -0,0 +1,158 @@
+const config = require('./config');
+
+const port = config.common.api.port || 8080;
+
+module.exports = {
+  openapi: '3.0.1',
+  info: {
+    version: '0.1.0',
+    title: '<%= projectName %>',
+    description: '<%= projectDescription %>',
+    termsOfService: '',
+    contact: {
+      name: 'Wolox',
+      email: 'tls@wolox.com.ar',
+      url: 'https://www.wolox.com.ar/'
+    },
+    license: {
+      name: 'MIT'
+    }
+  },
+  servers: [
+    {
+      url: `http://localhost:${port}/`,
+      description: 'Local server'
+    },
+    {
+      url: 'https://api_url_testing',
+      description: 'Testing server'
+    }
+  ],
+  security: [],
+  tags: [
+    {
+      name: 'CRUD operations'
+    }
+  ],
+  paths: {
+    '/users': {
```

Comment:
This file can grow too much, i know if we remove the example this would be simple, i was thinking in real documentation for a API and considering that this would be updated manually (at least in this version).
Can we modularize somehow ? Like a folder for schemas, other for paths and so

Reply:
I don't know about the `ref` behavior used to reference a component. I can try